sys-boot/raspberrypi-firmware/raspberrypi-firmware-1.20190215.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit readme.gentoo-r1
+
+DESCRIPTION="Raspberry Pi (all versions) bootloader and GPU firmware"
+HOMEPAGE="https://github.com/raspberrypi/firmware"
+SRC_URI="https://github.com/raspberrypi/firmware/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2 BSD"
+SLOT="0"
+KEYWORDS="-* ~arm ~arm64"
+
+S="${WORKDIR}/firmware-${PV}"
+
+QA_PREBUILT="boot/*.elf"
+
+src_configure() {
+	:;
+}
+
+src_compile() {
+	:;
+}
+
+src_install() {
+	insinto /boot
+	doins boot/*.{bin,dat,dtb,elf} "${FILESDIR}"/{cmdline,config}.txt
+
+	cat <<-EOF > "${T}"/99-raspberrypi-firmware || die
```

Comment:
This isn't wrong and it's not a showstopper, but I'd like to make you aware that you don't have to use `${T}` in here. 
```
newenvd - "99${PN}.conf" <<- _EOF_
	CONFIG_PROTECT="/boot/config.txt /boot/cmdline.txt"
_EOF_
```

no need for die etc. Keep in mind for future :)

Reply:
Sounds good.